src/route.rs

```diff
@@ -41,6 +41,7 @@ type BoxedRouteNewService<Req, Res> = Box<
 /// If handler is not explicitly set, default *404 Not Found* handler is used.
 pub struct Route {
     service: BoxedRouteNewService<ServiceRequest, ServiceResponse>,
+    #[allow(clippy::rc_buffer)]
     guards: Rc<Vec<Box<dyn Guard>>>,
```

Comment:
Could we add a comment to describe why this is allowed?

Reply:
So for one of these `guards` in this file, there is a `Rc::get_mut` usage, which basically invalidates this lint. The other ones are copied from / to that one. Not sure how to put that into a succinct commit. Maybe just move it to the top of the file as

```rust
#![allow(clippy::rc_buffer)] // because of Rc::get_mut
```
?

Reply:
Yeah, it would be helpful!